lib/Search/UI/templates/search.form.tpl

```diff
@@ -35,32 +35,41 @@
  * reasonably feasible for technical reasons, the Appropriate Legal Notices must
  * display the words "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  *}
-<h1 class="module-title-text">Search</h1>
+<h1 class="module-title-text hide">Search</h1>
```

Comment:
Why is it hidden, can you please remove it then?

Reply:
I hide this for Screen Readers.